Topic recognition

pandaarea opened the pull request.

pandaarea committed:
Test

A test file create a new folder

pandaarea committed:
Data load

data load and preprocessing

pandaarea committed:
Create README.md

pandaarea committed:
Add files via upload

pandaarea committed:
Update README.md

pandaarea committed:
Update README.md

pandaarea committed:
Update README.md

wooboyeong commented:
Code
* Model not implemented at all - only data loading

Commits
* Commit messages not informative

README
* README not informative - no description, no results at all

Pull Request
* No comments in the PR

shakes76 commented:
## TF/Torch Usage
No model implemented -5
## Good Practice (Design/Commenting)
Good spacing and comments
No Header blocks -1
## Algorithm 
Driver Script present
No Usage & Demo & Visualisation & Data usage -3
Module NOT present -1
No Data leakage
Difficulty: Easy -10
## Commit Log
Meaningful commit messages, could be more descriptive -1
Progressive commits used
## Documentation
Readme missing key components -5
Markdown used PDF submitted
## Pull Request
Successful Pull Request (Working Algorithm Delivered on Time in Correct Branch) 
Incomplete submission, cannot be merged -2
Request Description missing -2

pandaarea committed:
new project file

Schools' computers with GPU are currently unavailable in RDP. Restart the project in google colab.

pandaarea committed:
Data preprocessing

Split the dataset into training set, test set and validation set, and normalize the data

pandaarea committed:
Delete improved_unet.ipynb

pandaarea committed:
Add files via upload

Meet some problems when using colab, like slow speed and cannot running background. Reuse my own pc to run the project. 
Data preprocessing, and writing functions related to the improved unet model.

pandaarea committed:
Add files via upload

Optimized data preprocessing and increased the proportion of training set.
Limited by GPU memory, change images size to 192x192.
Added dice coefficient and dice coefficient loss functions.
The improved model was constructed and initially trained (started constructing the model on Friday, debugged the model to a runnable state today).

pandaarea committed:
Add files via upload

Optimize the program readability, divide the program into a model section and a driver section. 
Resize the image to (192, 256) according to the image size.
Constructed a plot function to compare predicted labels with original labels
Constructed a plot function to plot dice coefficient and dice loss curves.

pandaarea committed:
Add files via upload

Re-uploaded the driver file and module file

pandaarea committed:
Add files via upload

pandaarea committed:
Update README.md

pandaarea committed:
Update README.md

pandaarea committed:
Update README.md

pandaarea committed:
Update README.md

pandaarea committed:
Delete Project_1.ipynb

pandaarea committed:
Delete Test

pandaarea committed:
Delete Driver.ipynb

pandaarea committed:
Delete improved_unet_2.ipynb

pandaarea committed:
Add files via upload

Re-upload driver file and module file.

pandaarea committed:
Add files via upload

Integrate the code for processing data in a function.
Add module.py and driver.py files, these .py files are the latest version.

pandaarea committed:
Update README.md

pandaarea committed:
Update README.md

shakes76 commented:
Changes and feedback acknowledged from supplementary assessment.
